packages/fxa-auth-db-mysql/lib/db/schema/patch-099-100.sql

```diff
@@ -0,0 +1,22 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('99');
+
+CREATE PROCEDURE `readSecurityEvents_1` (
```

Comment:
See the note above about renaming this to `fetchSecurityEventsByUid_1` for clarity and to easily distinguish it from the existing `fetchSecurityEvents`